src/main/java/seedu/address/logic/commands/AddNumberOfRoomsCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic.commands;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Adds the number of hotel rooms in a hotel
+ */
+public class AddNumberOfRoomsCommand extends Command {
+    public static final String COMMAND_WORD = "addnumberofrooms";
+    private int numOfRooms;
+    private String numOfRoomsInString;
+
+    /**
+     * Creates an AddCommand to add the number of rooms available in a hotel
+     */
+    public AddNumberOfRoomsCommand(int numOfRooms) {
+        this.numOfRooms = numOfRooms;
+        this.numOfRoomsInString = "" + numOfRooms;
```

Comment:
hmm, why do you need this variable? Essentially, the ``numofRoomsInString``  is equal to ``Integer.toString(numOfRooms)``. so maybe you can just use that instead of declaring another variable?

Reply:
Okay I will change accordingly!